downstream/modules/platform/proc-gw-delete-roles.adoc

```diff
@@ -4,14 +4,10 @@
 
 = Deleting a role
 
-Built-in roles cannot be deleted, however, you can delete custom roles from the *Roles* list view.
+Default roles cannot be deleted; however, you can delete custom roles from the *Roles* list view.
 
 .Procedure
 
 . From the navigation panel, select {MenuAMRoles}.
-// [[hherbly]This may need to be replaced with updated steps for 2.6.]. Select a tab for the component resource for which you want to create custom roles.
-+
-// include::snippets/snip-gw-roles-note-multiple-components.adoc[]
-+
 . Click the *More Actions* icon *{MoreActionsIcon}* next to the role you want and select *Delete role*.
```

Comment:
Suggesting to use the method in the Accessibility spreadsheet and across various docs for the More Actions icon: 
```suggestion
. Click the btn:[More Actions] icon *{MoreActionsIcon}* next to the role you want and select *Delete role*.
```